example-sdl/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+set(OpenGL_GL_PREFERENCE LEGACY)
```

Comment:
Wouldn't it be better to update the OpenGL dependency to use GLVND libraries instead? Or did the API change too? Only the SDL example requires that.